ami-build/azp-minimal-x64.pkr.hcl

```diff
@@ -0,0 +1,51 @@
+# Defines a Packer template that builds an AMI image of a minimal x64 VM that runs
+# the AZP Agent.
+
+source "amazon-ebs" "envoy-azp-minimal-x64" {
+  ami_name = "envoy-azp-minimal-x64-{{timestamp}}"
+  instance_type = "r5.large"
+  region = "us-east-2"
+  security_group_ids = ["sg-030a7a75a086f208c"]
+
+  source_ami_filter {
+    filters = {
+      # Found with:
+      # aws ec2 describe-images --owners 'aws-marketplace' --output json --region us-east-2 --filters "Name=product-code,Values=4s6b2r2vfe46kyul508kf459f"
+      name = "ubuntu-minimal/images/hvm-ssd/ubuntu-jammy-22.04-amd64-minimal-*"
+      root-device-type = "ebs"
+      virtualization-type = "hvm"
+    }
+    most_recent = true
+    owners = ["099720109477"]
```

Comment:
Does this work? I think the owner ID of the minimal image is `679593333241` (confirmed with the command above).

Reply:
copypasta, given it didnt break anything iim wondering whether to remove or correct

Reply:
Weird, I expected it to act like a filter. If it doesn't work, we can remove it.

Might be better to figure out why, as this is supposed to protect us from loading an image with the same name, but from another source. We can figure this out in a future PR.

Reply:
ill comment it out with a comment for now